<launch>
  <node pkg="usb_cam" type="usb_cam_node" name="left">
    <param name="~camera_name" value="left"/>
    <param name="~camera_frame_id" value="left_cam"/>
    <param name="~pixel_format" value="mjpeg"/>
    <param name="~video_device" value="/dev/video2"/>
  </node>
</launch>
